fendermint/app/src/cmd/objects.rs

```diff
@@ -71,28 +70,23 @@ cmd! {
                 }
 
                 let client = FendermintClient::new_http(tendermint_url, None)?;
-
-                let iroh_addr = iroh_addr
-                    .to_socket_addrs()?
-                    .next()
-                    .ok_or(anyhow!("failed to convert iroh_addr to a socket address"))?;
-                let iroh_client = iroh::client::Iroh::connect_addr(iroh_addr).await?;
+                let iroh_manager = IrohManager::from_addr(Some(iroh_addr));
```

Comment:
use the iroh manager here so we don't have to worry about iroh being available when starting the objects API